xla/service/gpu/gemm_algorithm_picker_test.cc

```diff
@@ -60,15 +61,32 @@ class GemmAlgorithmPickerTest : public HloTestBase,
     return backend().default_stream_executor()->GetDeviceDescription();
   }
 
-  void SetUp() override {
-    const auto& gpu_cc = device_desc().gpu_compute_capability();
+  se::StreamExecutor *stream_exec() {
+    return backend().default_stream_executor();
+  }
+  const se::DeviceDescription& gpu_device_desc() {
+    return stream_exec()->GetDeviceDescription();
+  }
+  const se::GpuComputeCapability& gpu_comp() {
+    return gpu_device_desc().gpu_compute_capability();
+  }
 
-    if (auto* procm = std::get_if<se::RocmComputeCapability>(&gpu_cc)) {
-      if (GetDebugOptionsForTest().xla_gpu_enable_cublaslt() &&
-          !procm->has_hipblaslt()) {
-        GTEST_SKIP() << "No gpublas-lt support on this architecture!";
-      }
-    }
+  void SetUp() override {
+    std::visit(VariantVisitor{
+      [](const se::CudaComputeCapability& cc) {
```

Comment:
Putting this in `SetUp` will cause all tests to be skipped on Ampere+. However, currently the `BlasGetVersion` test is not skipped for `Ampere`. Please make sure to preserve the original behavior.